build/build.sh

```diff
@@ -27,6 +27,7 @@ dotnetCoreSdkDir=""
 function Help() {
   echo "Common settings:"
   echo "  -configuration <value>  Build configuration Debug, Release"
+  echo "  -host <value>           core (default), mono"
```

Comment:
nit: in the .ps1 we call it `hostType`. It would be nice if the parameter names match.